.github/workflows/code_scan.yaml

```diff
@@ -109,12 +109,13 @@ jobs:
         uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
         with:
           python-version: "3.10"
+      - name: Install uv
```

Comment:
Can we use the github action version instead? https://docs.astral.sh/uv/guides/integration/github/#installation